install_compatibility_layer.sh

```diff
@@ -10,7 +10,7 @@ REPOSITORY="software.eessi.io"
 RESUME=
 RETAIN_TMP=0
 STORAGE=
-VERSION=2023.06
+VERSION=2025.01
```

Comment:
```suggestion
VERSION=2025.02

Reply:
This should actually not be set at all, and this is changed in https://github.com/EESSI/compatibility-layer/pull/213/files#diff-f5a03e07ed775318214cf3eea39c31c6b589b82c8069834019ab92e497cb7e2a.